webapp/src/app/admin/teams/table/teams-table.component.ts

```diff
@@ -0,0 +1,241 @@
+import { SelectionModel } from '@angular/cdk/collections';
+import { Component, TrackByFunction, computed, effect, inject, input, signal } from '@angular/core';
+import { toObservable, toSignal } from '@angular/core/rxjs-interop';
+import { FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { lucideArrowUpDown, lucideChevronDown, lucideMoreHorizontal, lucideRotateCw, lucideXOctagon, lucidePlus, lucideCheck } from '@ng-icons/lucide';
+import { HlmButtonModule } from '@spartan-ng/ui-button-helm';
+import { HlmIconComponent, provideIcons } from '@spartan-ng/ui-icon-helm';
+import { HlmInputDirective } from '@spartan-ng/ui-input-helm';
+import { BrnMenuTriggerDirective } from '@spartan-ng/ui-menu-brain';
+import { HlmMenuModule } from '@spartan-ng/ui-menu-helm';
+import { BrnTableModule, PaginatorState } from '@spartan-ng/ui-table-brain';
+import { HlmTableModule } from '@spartan-ng/ui-table-helm';
+import { BrnSelectModule } from '@spartan-ng/ui-select-brain';
+import { HlmSelectModule } from '@spartan-ng/ui-select-helm';
+import { HlmSkeletonModule } from '@spartan-ng/ui-skeleton-helm';
+import { HlmCardModule } from '@spartan-ng/ui-card-helm';
+import { debounceTime, map } from 'rxjs';
+import { AdminService, TeamInfo } from '@app/core/modules/openapi';
+import { injectQueryClient } from '@tanstack/angular-query-experimental';
+import { octNoEntry } from '@ng-icons/octicons';
+import { HlmPopoverModule } from '@spartan-ng/ui-popover-helm';
+import { BrnPopoverComponent, BrnPopoverContentDirective, BrnPopoverTriggerDirective } from '@spartan-ng/ui-popover-brain';
+import { GithubLabelComponent } from '@app/ui/github-label/github-label.component';
+import { HlmScrollAreaComponent } from '@spartan-ng/ui-scrollarea-helm';
+
+const LOADING_TEAMS: TeamInfo[] = [
```

Comment:
If it errors I see this:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/55915b40-62c6-4ea8-a6bf-944f5dffa269">